docs/site-readme.md

```diff
@@ -1,154 +1 @@
-* Observe service status from a `ClowdApp` `status`, based on sevice dependencies.
-* Istio integration
-* Advanced logging configurations (e.g. logging messages to kafka topics)
-
-## Getting Clowder
-
-**Clowder is already running in pre-prod/prod environments.**
-
-To run Clowder locally in Minikube, obtain and install
-[Minikube](https://minikube.sigs.k8s.io/docs/start/).
-
-Clowder is developed on Fedora and the kvm driver has been found to work best
-initiated with the following options:
-
-```shell
-minikube start --cpus 4 --disk-size 36GB --memory 16000MB --driver=kvm2 --addons registry --addons ingress  --addons=metrics-server --disable-optimizations
-```
-
-NOTE:
-Mac OS is also supported with the `virtualbox` and `hyperkit` drivers. A full
-guide [can be found here](docs/macos.md)
-
-To persist these changes for every minikube invocation, run the following:
-
-```shell
-minikube config set cpus 4
-minikube config set memory 16000
-minikube config set disk-size 36GB
-minikube config set driver kvm2
-```
-
-If you encounter any kvm issues, please take a look
-[at the troubleshooting guide](docs/developer-guide.md)
-
-The ``kube_setup.sh`` script then needs to be run by invoking
-
-```shell
-./build/kube_setup.sh
-```
-
-Clowder can then be installed by running:
-
-```shell
-# Be sure to get the latest release in the link above!
-minikube kubectl -- apply -f $(curl https://api.github.com/repos/RedHatInsights/clowder/releases/latest | jq '.assets[0].browser_download_url' -r) --validate=false
-```
-
-## Usage
-
-To use Clowder to deploy an application a ``ClowdEnvironment`` resource must be
-present to define an environment. Once this has been deployed, a ``ClowdApp``
-resource is authored for the app and deployed alongside the ``ClowdEnvironment``.
-
-Example app developer workflow:
-
-* Install Clowder on a minikube environment.
-* Use ``kubectl apply -f clowdenv.yaml`` to apply a ``ClowdEnvironment`` resource
-  to the cluster.
-* Use ``kubectl apply -f clowdapp.yaml`` to apply a ``ClowdApp`` resource to the
-  cluster.
-
-More details on how to do this are present in the [Getting Started](docs/usage/getting-started.md) section
-of the documentation.
-
-[API Reference](https://redhatinsights.github.io/clowder/clowder/dev/api_reference.html)
-
-## Building Clowder
-
-If you want to run a version of Clowder other than the released version there
-are a few prerequisites you will need. To learn about developing Clowder please
-visit the [developing clowder](docs/developer-guide.md) page for more detailed instructions.
-
-## History
-
-To understand more about the design decisions made while developing Clowder,
-please visit the [design document](docs/clowder-design.adoc).
-
-## Connect
-
-Any questions, please ask one of the Clowder development team
-
-* [@psav](https://github.com/psav)
-* [@bsquizz](https://github.com/bsquizz)
-* [@bennyturns](https://github.com/bennyturns)
-* [@adamrdrew](https://github.com/adamrdrew)
+docs/site-readme.md
```

Comment:
What is this change doing? - And why is it in this PR :)

Reply:
i have no idea how that got there, restored the file